demo/app/lib/views/supported_credentials_list.dart

```diff
@@ -51,31 +51,47 @@ class SupportedCredentialsListState extends State<SupportedCredentialsList> {
               padding: const EdgeInsets.fromLTRB(12, 12, 12, 12),
               itemCount: widget.supportedCredentialList.length,
               itemBuilder: (context, index) {
-                return Card(
-                  shape: RoundedRectangleBorder(
-                    borderRadius: BorderRadius.circular(12),
-                  ),
-                  color: Colors.grey.shade200,
-                  clipBehavior: Clip.antiAliasWithSaveLayer,
+                return Container(
+                  height: 80,
+                  alignment: Alignment.center,
+                  decoration: BoxDecoration(
+                      color: '0xff${widget.supportedCredentialList.elementAt(index).display.elementAt(index).backgroundColor.toString().replaceAll('#', '')}'
+                              .isNotEmpty
+                          ? Color(int.parse('0xff${widget.supportedCredentialList.elementAt(index).display.elementAt(index).backgroundColor.toString().replaceAll('#', '')}'))
+                          : Colors.white,
+                      borderRadius: BorderRadius.circular(12),
+                      boxShadow: [BoxShadow(offset: const Offset(3, 3), color: Colors.grey.shade300, blurRadius: 5)]),
                   child: ListTile(
-                    title: widget.supportedCredentialList.elementAt(index).types[index] == "VerifiableCredential"
-                        ? Text(
-                            widget.supportedCredentialList.elementAt(index).types[index + 1],
-                            style: const TextStyle(
-                              fontSize: 14,
-                              color: Color(0xff190C21),
-                            ),
-                          )
-                        : null,
-                    leading: IconButton(
-                      icon: const Icon(Icons.credit_card_sharp),
-                      tooltip: 'Credential',
-                      onPressed: () {},
+                    title: Text(
+                      widget.supportedCredentialList.elementAt(index).display.elementAt(index).name,
+                      style: TextStyle(
+                        fontSize:14,
+                        fontWeight: FontWeight.bold,
+                        color: '0xff${widget.supportedCredentialList.elementAt(index).display.elementAt(index).textColor.toString().replaceAll('#', '')}'
+                                .isNotEmpty
```

Comment:
this construction `supportedCredentialList.elementAt(index).display.elementAt(index)` looks realy wiered.  Each credential in this credential list contains display data for all other credentials. In another case, it will not work. 

Also, the type of supportedCredentialList is List<SupportedCredentials>. But SupportedCredentials already list of credentials. So that wrapping list is not needed. Or `SupportedCredentials` should be changed to `SupportedCredential` and contains data only for one credential.

Reply:
@DRK3  Please confirm the display list is an array itself does it contain display for all other supported credentials as well ? In my opinion it only contains display for the specific supported credential 

```
 [
{
 format: jwt_vc_json, 
types: [VerifiableCredential, VerifiedEmployee], 
display: [
 { 
name: Verified Employee,
 locale: en-US, 
logo: logo.png, 
textColor: #FFFFFF, 
backgroundColor: #0000FF 
}
] 
}
]
```